test/uk/gov/hmrc/tradergoodsprofiles/controllers/MaintainProfileControllerSpec.scala

```diff
@@ -99,6 +104,21 @@ class MaintainProfileControllerSpec extends PlaySpec with AuthTestSupport with B
       status(result) mustBe OK
       contentAsJson(result) mustBe Json.toJson(updateProfileResponse)
     }
+    // TODO: Create a single test - Ticket-2014
+    "return 200 OK without validating x-client-id when isDrop1_1_enabled is true" in {
+      when(appConfig.isDrop1_1_enabled).thenReturn(true)
```

Comment:
minor, but test structure usually follow

Given (your object set up) 
When (your mocking and function call) 
Then (your assertions)

So in this case, the when statement should be after the fake request

Reply:
You're right! Done!